sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/library/build.sbt

```diff
@@ -8,13 +8,6 @@ npmDependencies in Compile += "node-uuid" -> "1.4.7"
 
 //#relevant-settings
 webpackConfigFile := Some(baseDirectory.value / "webpack.config.js")
-
-import org.scalajs.core.tools.io.{VirtualJSFile, FileVirtualJSFile}
-
-// Use the output of Scala.js as a “launcher”
-scalaJSLauncher in (Compile, fullOptJS) := {
-  Attributed.blank[VirtualJSFile](FileVirtualJSFile((fullOptJS in Compile).value.data))
-}
```

Comment:
I'm running blind right now because `manual/previewSite` doesn't work on Windows. I think it should be fine, though.